pkg/scheduler/factory.go

```diff
@@ -455,28 +455,28 @@ func NewPodInformer(client clientset.Interface, resyncPeriod time.Duration) core
 func MakeDefaultErrorFunc(client clientset.Interface, podQueue internalqueue.SchedulingQueue, schedulerCache internalcache.Cache) func(*framework.PodInfo, error) {
 	return func(podInfo *framework.PodInfo, err error) {
 		pod := podInfo.Pod
+		_, isFitError := err.(*core.FitError)
```

Comment:
This is not a good one IMO.

Reply:
Thx, I will fix it later.

Reply:
Like this?
```
else if _, ok := err.(*core.FitError); ok {
}
```
I had asked a friend, he suggested me.

Reply:
> Like this?
> 
> ```
> else if _, ok := err.(*core.FitError); ok {
> }
> ```
> 
> I had asked a friend, he suggested me.

Yes

Reply:
A shorter lifecycle var is welcome, I got it. Thx

Reply:
> > Like this?
> > ```
> > else if _, ok := err.(*core.FitError); ok {
> > }
> > ```
> > 
> > 
> > I had asked a friend, he suggested me.
> 
> Yes

A shorter lifecycle var is welcome, I got it. Thx